### DISCUSSION TITLE
GITHUB API - Sort by "id" - Search repository

### DISCUSSION THREAD

**Discussion by gabrieldeoliveiraestevam** [community/community#24747] [Apps, API and Webhooks]
<p>Guys,</p>
<p>How to sort by repository id?</p>
<p>I’m trying to make the following request. However, it is not sorting by “id”:</p>
<p>“<a href="https://api.github.com/search/repositories?q=topic:data-science&amp;sort=id&amp;order=desc&amp;page=1&amp;per_page=100" rel="noopener nofollow ugc">https://api.github.com/search/repositories?q=topic:data-science&amp;sort=id&amp;order=desc&amp;page=1&amp;per_page=100</a>”</p>

**Comment by parithiban** [comment, marked as answer]
<p><a class="mention" href="/gabrieldeoliveiraestevam">@gabrieldeoliveiraestevam</a></p>
<p>Currently there is no sort option available for <code>id</code> parameter. Kindly have a look on <a href="https://docs.github.com/en/github/searching-for-information-on-github/getting-started-with-searching-on-github/sorting-search-results" class="inline-onebox" rel="noopener nofollow ugc">Sorting search results - GitHub Docs</a>.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/3/436dd3265670696765cde6eebf6df51bcc5f5cc6.png" data-download-href="/uploads/short-url/9CvhVd7uwnJR8mwaxut402aIWwu.png?dl=1" title="Screenshot 2021-06-07 at 7.20.00 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/7281166/181086554-8ac8d948-a325-4edd-874e-1373868be09a.png" alt="Screenshot 2021-06-07 at 7.20.00 PM" data-base62-sha1="9CvhVd7uwnJR8mwaxut402aIWwu" width="690" height="360" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/3/436dd3265670696765cde6eebf6df51bcc5f5cc6_2_690x360.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/3/436dd3265670696765cde6eebf6df51bcc5f5cc6_2_1035x540.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/4/3/436dd3265670696765cde6eebf6df51bcc5f5cc6_2_1380x720.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/3/436dd3265670696765cde6eebf6df51bcc5f5cc6_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2021-06-07 at 7.20.00 PM</span><span class="informations">2878×1502 580 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
